templates/releases/activity.html

```diff
@@ -28,27 +28,21 @@
         new Chart(ctx, {
             type: "line",
             data: {
-                labels: [
-                    {% if activity_data.dates -%}
-                        {%- for date in activity_data.dates -%}
-                            {{ "'" ~ date ~ "'," }}
-                        {%- endfor -%}
-                    {%- endif %}
-                ],
+                labels: {{ dates | json_encode() | safe }},
```

Comment:
I'm pretty sure | safe is on by default.

Reply:
`| safe` is really `unsafe`, it says to disable html escaping.